src/sentry/integrations/github/client.py

```diff
@@ -103,6 +104,27 @@ def get_tree(self, repo_full_name: str, tree_sha: str) -> List[str]:
 
         return tree
 
+    def get_repo_files(
+        self, repo_full_name: str, tree_sha: str, only_source_code_files: bool = True
+    ) -> List[str]:
+        """It return all files for a repo or just source code files.
+
+        repo_full_name: e.g. getsentry/sentry
+        tree_sha: A branch or a commit sha
+        only_source_code_files: Include all files or just the source code files
+        """
+        repo_files = []
+        try:
+            tree = self.get_tree(repo_full_name, tree_sha)
+            if tree is not None:
+                repo_files = [x["path"] for x in tree if x["type"] == "blob"]
+                if only_source_code_files:
```

Comment:
For now, we only use this function with the default value but we may in the future call it with a False value.